docs/deployment.rst

```diff
@@ -96,23 +96,24 @@ Next we need to configure *aiohttp upstream group*:
 
    http {
      upstream aiohttp {
-       fail_timeout=0 means we always retry an upstream even if it failed
+       # fail_timeout=0 means we always retry an upstream even if it failed
```

Comment:
That would be easy enough to do. I can commit a version like this and we'll evaluate if it's too verbose.